orttraining/orttraining/training_ops/cuda/collective/nccl_kernels.cc

```diff
@@ -219,6 +249,7 @@ ONNX_OPERATOR_KERNEL_EX(
     kCudaExecutionProvider,
     KernelDefBuilder()
         .Alias(AliasRange(0, 1024))
+        .AllocateInputsContiguously()
```

Comment:
>AllocateInputsContiguously [](start = 9, length = 26)

This is a nice trick! BTW I have concerns if 1024 is going to be enough for big models...

Reply:
:)

Reply:
Jesse's trick! He created it when he protyped continuous allocation for statically allocated activations.  

---
In reply to: [453883338](https://github.com/microsoft/onnxruntime/pull/4443#discussion_r453883338) [](ancestors = 453883338)